docs/paper.md

```diff
@@ -55,10 +55,11 @@ In addition, ASTE provides performance and accuracy metrics of the configured si
 Besides preCICE itself, core ingredients for practical applications are preCICE API language bindings, preCICE adapters, the simulation frameworks and their dependencies.
 ASTE, on the other hand, replaces coupled models and only requires a reduced set of dependencies.
 It abstracts the computational complexity of the models away by extracting the relevant information from VTK files instead and passing extracted data to preCICE, potentially in parallel on distributed memory.
-While the VTK files may stem from actual simulations, ASTE can also generate artificial VTK files with prescribed coupling data.
+VTK files can be produced from existing preCICE simulations without recompilation.
```

Comment:
Why should there be any recompilation?

Reply:
I try to make sure that using this feature isn't a big pain point.
It is common to `#ifdef`remove such output when applications don't use it. Similar to the PoU cluster export.

Maybe:
```suggestion
VTK files can be produced from existing preCICE simulations by enabling exports.
```

Reply:
Isn't this clear from the sentence below?

> The VTK files can be generated from mesh generation tools (e.g., GMSH [@gmsh]), [Python scripts](https://github.com/precice/aste/tree/develop/tools/mesh-generators) or directly reused from a [completed preCICE simulation](https://precice.org/configuration-export.html).

Reply:
I'll rewrite the suggestion. Sec